sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -706,7 +706,7 @@ object ShowTables {
 case class ShowTableExtended(
     namespace: LogicalPlan,
     pattern: String,
-    partitionSpec: Option[PartitionSpec],
+    partitionSpec: Option[TablePartitionSpec],
```

Comment:
Please, use the type `PartitionSpec`. The `TablePartitionSpec` is for v1 commands, and shouldn't be used in v2 commands.

Reply:
Ok, done!